### ISSUE TITLE
integration flake: Timed out waiting for apiserver

### ISSUE THREAD

**Issue by pmorie** [kubernetes/kubernetes#18713]
I've seen several PRs hit this in the last 2 days or so.  Example:

https://console.developers.google.com/storage/browser/kubernetes-jenkins/pr-logs/5e0948586c35e7e7f4f7ccc39411d2726c8e767a/kubernetes-pull-test-unit-integration/5875/

```
!!! [1214 19:06:49] Timed out waiting for apiserver:  to answer at http://127.0.0.1:8080/healthz; tried 25 waiting 0.5 between each
!!! Error in ./hack/test-update-storage-objects.sh:48
  'return 1' exited with status 1
Call stack:
  1: ./hack/test-update-storage-objects.sh:48 startApiServer(...)
  2: ./hack/test-update-storage-objects.sh:122 main(...)
Exiting with status 1
```

**Comment by wojtek-t** [comment]
Did we see it since #18814 was merged?

**Comment by wojtek-t** [comment]
I think that it was fixed by increasing the timeout. Please reopen this issue if you observe it again.

**Comment by mml** [comment]
This is happening again.  See investigation in #19435

**Comment by spxtr** [comment]
```
I0121 19:11:40.691764    4739 genericapiserver.go:325] Node port range unspecified. Defaulting to 30000-32767.
[restful] 2016/01/21 19:11:40 log.go:30: [restful/swagger] listing is available at https://172.17.0.3:6443/swaggerapi/
[restful] 2016/01/21 19:11:40 log.go:30: [restful/swagger] https://172.17.0.3:6443/swaggerui/ is mapped to folder /swagger-ui/
!!! [0121 19:13:45] Timed out waiting for apiserver:  to answer at http://127.0.0.1:8080/healthz; tried 120 waiting 1 between each
!!! Error in ./hack/test-update-storage-objects.sh:48
  'return 1' exited with status 1
Call stack:
  1: ./hack/test-update-storage-objects.sh:48 startApiServer(...)
  2: ./hack/test-update-storage-objects.sh:123 main(...)
Exiting with status 1
```
#19926

**Comment by davidopp** [comment]
AFAICT #19435 is an exact dupe of this issue. @mml was there some reason you opened a separate issue? I'm going to close this one as the other one has more debugging info.
